src/ape/managers/converters.py

```diff
@@ -142,25 +142,37 @@ def convert(self, value: Union[List, Tuple]) -> Union[List, Tuple]:
 
 class TimestampConverter(ConverterAPI):
     """
-    Converts string of ``"%m-%d-%Y %H:%M:%S"`` format to a timestamp.
+    Converts either a string of ``"%m-%d-%Y %H:%M:%S"`` format to a timestamp OR
+    converts a datetime object to a timestamp.
     Must be UTC timezone
     """
 
-    def is_convertible(self, value: str) -> bool:
-        if not isinstance(value, str):
+    def is_convertible(self, value: Union[str, datetime]) -> bool:
+        if not isinstance(value, str) and not isinstance(value, datetime):
             return False
-        if " " not in value or len(value.split(" ")) > 2:
-            return False
-        try:
-            datetime.strptime(value, "%m-%d-%Y %H:%M:%S")
-        except ValueError:
+        if isinstance(value, str):
+            if " " not in value or len(value.split(" ")) > 2:
+                return False
+            else:
+                try:
+                    datetime.strptime(value, "%m-%d-%Y %H:%M:%S")
+                except ValueError:
+                    return False
+        if isinstance(value, datetime) and str(value.tzinfo) != "UTC":
             return False
         return True
 
-    def convert(self, value: str) -> int:
-        return int(
-            datetime.strptime(value, "%m-%d-%Y %H:%M:%S").replace(tzinfo=timezone.utc).timestamp()
-        )
+    def convert(self, value: Union[str, datetime]) -> int:
+        if isinstance(value, str):
+            return int(
+                datetime.strptime(value, "%m-%d-%Y %H:%M:%S")
+                .replace(tzinfo=timezone.utc)
+                .timestamp()
+            )
+        elif isinstance(value, datetime):
+            return int(value.timestamp())
+        else:
+            raise ConversionError
```

Comment:
You can probably get rid of this `else` and just make the `elif` an else.

We should never get here with any other types because of the `is_convertible()` method.

I would say it is safe to assume that the given value is either a `str` or a `datetime`